src/components/dialog/faultEvent/FaultEventCreation.tsx

```diff
@@ -168,20 +168,18 @@ const FaultEventCreation = ({ useFormMethods, isRootEvent }: Props) => {
             />
 
             {/* Probability field */}
-            {eventTypeWatch !== EventType.INTERMEDIATE && (
-              <TextField
-                label={t("newFtaModal.probability")}
-                type="number"
-                min={0}
-                max={1}
-                step={0.01}
-                error={!!errors.probability}
-                helperText={errors.probability?.message}
-                className={classes.probability}
-                defaultValue=""
-                {...register("probability")}
-              />
-            )}
+            <TextField
```

Comment:
@LaChope I do not understand this. Why would you want to set up probability for intermediate event ?

Reply:
@LaChope you resolved this but did not provide info how

Reply:
We resolved it during Stand-up but yes I did not write it here: We agreed that Intermediate and external events should not have probability, all other event types should have one

Reply:
Resolved by https://github.com/kbss-cvut/fta-fmea-ui/pull/396.